### PR TITLE
[prakriya] Add transliterator to debugger

### PR DESCRIPTION
**Pull request by ma08**
Add transliterate feature (issue #42) for `vidyut-prakriya` demo using a dropdown selection
Currently supporting:
devanagari, iast, slp1, itrans, telugu, kannada

Fix couple of bugs in running debugger

### CODE REVIEW

**Comment by ma08** [comment]
@akprasad Not a pressing issue but we might have to decide if we want to transliterate headings too (`तिङन्तानि` `कृदन्तानि` `प्रयोगः` etc.)

File: vidyut-prakriya/www/templates/index.html [hunk 1]
```diff
@@ -255,6 +255,17 @@ <h1 class="text-4xl font-bold mb-2">{{ 'vidyullekhA'|d }}</h1>
     </p>
   </header>
 
+  <div class="max-w-3xl mx-auto text-right">
+    <select x-model="script" class="p-2 py-2 my-2">
```
Comment (akprasad) [inline vidyut-prakriya/www/templates/index.html:259]: you can remove `py-2` since it is included in `p-2`.

**Comment by akprasad** [comment]
Wonderful! Congrats on preparing your first PR.

I have one small comment -- otherwise it looks good to me!

**Review by akprasad — APPROVED** [review]

File: vidyut-prakriya/www/templates/index.html (hunk 1 above)
Comment (ma08) [reply vidyut-prakriya/www/templates/index.html:259]: Aah! I understand now, my css-fu isn't great. Good to know! Will make the change and push.

Comment (ma08) [reply vidyut-prakriya/www/templates/index.html]: @akprasad Have made the change and refactored the `wasm-pack` check as suggested Gouenji in discord.

**Pull request merged**